pulp_container/app/downloaders.py

```diff
@@ -32,7 +31,6 @@ def __init__(self, *args, **kwargs):
         self.remote = kwargs.pop("remote")
         super().__init__(*args, **kwargs)
 
-    @backoff.on_exception(backoff.expo, ClientResponseError, max_tries=10, giveup=http_giveup)
     async def _run(self, handle_401=True, extra_data=None):
         """
         Download, validate, and compute digests on the `url`. This is a coroutine.
```

Comment:
can you update the doc block https://github.com/pulp/pulpcore/pull/1401/files#diff-3b08c4799559edbce70a244edcc7bcc6e3817e9e8837da6e5519cbc0b3271feaR264